l10n_mx_sat_account/data/sat.account.csv

```diff
@@ -0,0 +1,1083 @@
+"id","name","code","parent_id/id"
+"sat_account_id1","Activo","100",
```

Comment:
Is not the same data from l10n_mx community module?
 - https://github.com/odoo/odoo/blob/7e5a7245e86d8a3bcdab83c740cb4d188d009a40/addons/l10n_mx/data/account_tag_data.xml#L5

What about just creating a constraint in order to avoid duplicated configuration?

Reply:
@moylop260 I am not sure to follow you here. What kind of constraint?

Reply:
I understood that tags is not a good approach because it is m2m field.
So It is fixed creating a new model duplicating the same records.

If there is a issue using m2m fields so you can make a constraint to avoid this one:

https://github.com/OCA/l10n-mexico/pull/12#discussion_r368258096

So you will avoid user confusing duplicated models and records since that l10n_mx is a module auto-installed after choose Mexico country.

Reply:
@moylop260 If I understand you well, you want to add a constraint on the tags to make sure there is only one "SAT" tag set per account. Correct?